extension/src/LanguageFunctions.ts

```diff
@@ -186,6 +186,55 @@ export async function copySourceToTarget(): Promise<boolean> {
   return false;
 }
 
+export async function copyAllSourceToTarget(
+  languageFunctionsSettings: LanguageFunctionsSettings,
+  setAsReview: boolean
+): Promise<boolean> {
+  if (vscode.window.activeTextEditor) {
+    if (vscode.window.activeTextEditor.document.uri.fsPath.endsWith("xlf")) {
+      // in a xlf file
+      const filePath = vscode.window.activeTextEditor.document.uri.fsPath;
+      await vscode.window.activeTextEditor.document.save();
```

Comment:
Move `vscode` calls upstream and only pass filepath maybe? Makes for a cleaner function :)

Reply:
That thought flew through my mind as well, but I just did what all other functions here did... 😁
But that would at least make it more testable, so I'll change this.

Reply:
The return true/false made it feel like a copy/paste of some function :sweat_smile:

Reply:
> The return true/false made it feel like a copy/paste of some function 😅

Speak of lazy dev, you know... 😉